client/src/pages/AdminDashboard.js

```diff
@@ -149,65 +191,85 @@ const AdminDashboard = (props) => {
-    for (let eventId of users.keys()) {
-      if(propName === 'eventType'){
+    uniqueTypes.forEach((el) => (result[el] = parseInt('0')));
+    for (const eventId of users.keys()) {
+      if (propName === 'eventType') {
         type = events.get(eventId).eventType;
         result[type] = users.get(eventId).length + result[type];
       }
 
-      if(propName === 'hacknight' && typeof events.get(eventId).hacknight !== 'undefined'){
+      if (
+        propName === 'hacknight' &&
+        typeof events.get(eventId).hacknight !== 'undefined'
+      ) {
         type = events.get(eventId).hacknight;
         result[type] = users.get(eventId).length + result[type];
       }
     }
     return result;
   }
 
-  function findTotalVolunteerHours(events, users, uniqueTypes, propName){
-    let result = {};
+  function findTotalVolunteerHours(events, users, uniqueTypes, propName) {
+    const result = {};
     let type;
-    uniqueTypes.forEach(el => result[el] = parseInt('0'));
+    uniqueTypes.forEach((el) => (result[el] = parseInt('0')));
 
-    for (let eventId of users.keys()) {
-      if(propName === 'eventType'){
+    for (const eventId of users.keys()) {
+      if (propName === 'eventType') {
         type = events.get(eventId).eventType;
-        result[type] = result[type] + (events.get(eventId).hours * users.get(eventId).length);
+        result[type] =
+          result[type] + events.get(eventId).hours * users.get(eventId).length;
       }
 
-      if (propName === 'hacknight' && typeof events.get(eventId).hacknight !== 'undefined'){
+      if (
+        propName === 'hacknight' &&
+        typeof events.get(eventId).hacknight !== 'undefined'
+      ) {
         type = events.get(eventId).hacknight;
-        result[type] = result[type] + (events.get(eventId).hours * users.get(eventId).length);
+        result[type] =
+          result[type] + events.get(eventId).hours * users.get(eventId).length;
       }
     }
     return result;
   }
 
-  function findAverageVolunteerHours(totalVolunteers, totalVolunteerHours, uniqueTypes){
-    let result = {};
-    uniqueTypes.forEach(el => result[el] = parseInt('0'));
-
-    for(let eventType of uniqueTypes.keys()){
-      let hours = totalVolunteerHours[eventType];
-      let volunteers = totalVolunteers[eventType];
-      let averageHours = (hours / volunteers);
-      if(!Number.isInteger(averageHours)) averageHours = +averageHours.toFixed(2);
-      volunteers > 0 ? result[eventType] = averageHours : result[eventType] = 0;
+  function findAverageVolunteerHours(
+    totalVolunteers,
+    totalVolunteerHours,
+    uniqueTypes
+  ) {
+    const result = {};
+    uniqueTypes.forEach((el) => (result[el] = parseInt('0')));
+
+    for (const eventType of uniqueTypes.keys()) {
+      const hours = totalVolunteerHours[eventType];
+      const volunteers = totalVolunteers[eventType];
+      let averageHours = hours / volunteers;
+      if (!Number.isInteger(averageHours))
```

Comment:
let's use brackets for such `if` statements

Reply:
I am not sure if that is workable syntax, but please let me know if I am missing something. I attached a screenshot to check on it. 

For my opinion on brackets vs parenthesis, it is just not my preference as well. I feel like it can misconstrue that an item is a loop versus being an evaluation statement. Those are my two cents though. 

![image](https://user-images.githubusercontent.com/13891656/93655037-e7c4b500-f9d5-11ea-8e8a-e47e35c18318.png)

Reply:
I made a mistake in my comment: braces, not brackets. Sorry about that. Here is an example:
```
if (condition) {
  statement
}
```
even if the statement is 1 line

Reply:
I think that this is a case of actually not looking too far into it. I am looking at the code and I think that it's broken. Here is the `development` branch with the same method and I am not sure what the intended goal of the lines at this time is. Do you have any thoughts on what these lines are intending to do @alex-anakin ? https://github.com/hackforla/VRMS/blob/development/client/src/pages/AdminDashboard.js#L190

In summary, I don't think that this is the formatter's issue, as much as code that is breaking prettier.

Reply:
I don't see anything broken in the code. The line that you mentioned is a function declaration.
```
function findAverageVolunteerHours(totalVolunteers, totalVolunteerHours, uniqueTypes) {
  // some calculations for component business logic
  return result;
}
```
What is wrong here for you?

I think you misunderstood my first comment. I want prettier to format this statement
`if (!Number.isInteger(averageHours)) averageHours = +averageHours.toFixed(2);`
to be the following:
```
if (!Number.isInteger(averageHours)) {
  averageHours = +averageHours.toFixed(2);
}
```

Regarding your comment about code that breaks the prettier I need to say that working code comes first formatting comes second.

Reply:
Yeah, sorry, I was referencing that line there,  `if (!Number.isInteger(averageHours)) averageHours = +averageHours.toFixed(2);`. That does not look to be proper syntax and looks to be breaking the autoformatter. Mystery  apologies for sending you the function that it housed in and not the line itself. I thought that I had, but it seems that I missed that. Below is why I think that the syntax doesn't make sense to me, but I also think that this is the reason it is breaking the formatter. 

![image](https://user-images.githubusercontent.com/13891656/93944907-b6f0c280-fcea-11ea-948c-1469e5a0c562.png)

I would still defer to you on this. Are you expecting that the value is reassigned to it's positive? Or do you think it has another intention?

Reply:
`if (!Number.isInteger(averageHours)) averageHours = +averageHours.toFixed(2);` 

There I determine that `averageHours` is not an integer. If true, convert it to a number of 2 digits after the decimal point. Method toFixed() returns a string representing a number in fixed-point notation. So, I converted it to a number using '+', because in total it will return NaN.

```
/**
 * Returns true if the value passed is an integer, false otherwise.
 * @param number A numeric value.
 */
isInteger(number: unknown): boolean;

/**
 * Returns a string representing a number in fixed-point notation.
 * @param fractionDigits Number of digits after the decimal point. Must be in the range 0 - 20, inclusive.
 */
toFixed(fractionDigits?: number): string;

 /**
  * Returns a supplied numeric expression rounded to the nearest integer.
  * @param x The value to be rounded to the nearest integer.
  */
 round(x: number): number;

```
If it's breaking the linter. The above line could be replaced with this one without a problem:

```
if (!Number.isInteger(averageHours)) {
  averageHours = Math.round(100 * averageHours) / 100;
}
```

Reply:
Yeah, this would work. Thanks @OlgaBilogurova, yeah this syntax is breaking the auto-formatter, so thanks for an alternative.

Reply:
Hey @OlgaBilogurova, does this look right? 
![image](https://user-images.githubusercontent.com/13891656/94387248-70da9b00-00fe-11eb-80c5-60dfbf6d55da.png)